docs/en/inference/convert_tutorial.md

```diff
@@ -166,13 +181,35 @@ paddle2onnx \
 The `input_shape_dict` in the parameter can generally be viewed by opening the inference model using the [Netron](https://github.com/lutzroeder/netron),
 or found in the code in [tools/export_model. py](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/tools/export_model.py) above.
 
-#### 2.3 ONNX -> OM
+#### 2.3 ONNX -> MindIR
+
+The `converter_lite` can be used to convert the ONNX into a MindSpore Lite MindIR. For detailed usage tutorials,
```

Comment:
The `converter_lite` script can be used to convert an ONNX model to a MindIR model.